CentralComputing/Sensor_Package.cpp

```diff
@@ -48,14 +52,19 @@ Sensor_Package::Sensor_Package(vector<Sensor_Configuration> configuration) {
 	//16-bit RideHeight3
 	Xmega_Setup x1 = {"/dev/spidev1.0", 3, bpi1, 500000, 8};//Xmega1 initialization struct
 	Xmega_Setup x2 = {"/dev/spidev1.1",16, bpi2, 500000, 8};
-
-	spi = new Spi(&x1, &x2);
+	
+	if(xmega_connect) {
```

Comment:
Would this be better as a `#ifdef` ? 
Only time SPI won't be used is during a testing scenario, right? And if it is a testing scenario, then `#define TEST_POD` is defined. 
What do you think? It is a small change, but why compile a code path that will NEVER be used during normal operation